File: LicentaFront/src/app/modules/product.ts

import { Overbidding } from "./overbidding";



export interface Product {
  id?: number;
  userId?: number;
  categoryId?: number;
  title: string;
  price: number;
  text?: string;
  overbidding?: Overbidding[];
  view?: number;
  startingDate?: Date;
  expirationDate?: Date;
  image: Blob,
}

File: LicentaApi/Models/Product.cs
namespace LicentaApi.Models
{
    public class Product
    {
        public long Id { get; set; }

        public long? UserId { get; set; }
        public string? Title { get; set; }
        public byte[]? Image { get; set; }

        public long? CategoryId { get; set; }

        public long? Price { get; set; }

        public string? Text { get; set; }

        public List<Overbidding>? Overbiddings { get; set; }

       

        public int? View {get; set; }

        public DateTime? StartingDate { get; set; }

        public DateTime? ExpirationDate { get; set; }

    }
}